float64 left_lid
float64 right_lid
---
string msg_feedback
